Source/WTF/wtf/spi/darwin/AbortWithReasonSPI.h

```diff
@@ -51,7 +50,7 @@ void abort_with_reason(uint32_t reason_namespace, uint64_t reason_code, const ch
 }
 #endif
 
-#if !USE(APPLE_INTERNAL_SDK)
+#if !USE(APPLE_INTERNAL_SDK) || (USE(APPLE_INTERNAL_SDK) && defined(SWIFT_ONLY))
```

Comment:
The second USE(APPLE_INTERNAL_SDK) isn't needed.
I'd call SWIFT_ONLY something like WK_COMPILING_SWIFT

Reply:
I think you can use `__swift__` to check whether the header is being read by swift's clang importer. So you don't need to define your own SWIFT_ONLY macro.

Reply:
renamed to WK_SWIFT_INTEROP_HEADER_GENERATION

Reply:
> I think you can use `__swift__` to check whether the header is being read by swift's clang importer. So you don't need to define your own SWIFT_ONLY macro.

Will this always be true and never break in future ?

Reply:
I mean, it's part of how the clang importer works and is used extensively in Swift's standard library: https://github.com/search?q=repo%3Aapple%2Fswift%20__swift__&type=code

So, yes, I think it is as stable as other compiler-provided macros like `__objc__`.